docs/proposals/registry/terminology.md

```diff
@@ -0,0 +1,23 @@
+# Devfile Registry Terminology
+
+The following terms may be used throughout design proposals for devfile registries
+
+## Devfile Index Image
+
+The container image containing the devfile stacks and index.json used to bootstrap the OCI registry with devfile stacks
```

Comment:
This image also contains the server for serving the registry index

Reply:
Updated.